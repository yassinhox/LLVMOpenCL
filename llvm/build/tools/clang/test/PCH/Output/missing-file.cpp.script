echo 'struct S{char c; int i; }; void foo() {}' > /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.h &&
echo 'template <typename T> void tf() { T::foo(); }' >> /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.h &&
/home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -x c++ -emit-pch -o /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.h.pch /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.h &&
rm /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.h || rm /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.h || rm /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.h &&
not /home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -include-pch /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.h.pch -emit-obj -o /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.o /home/jazouani/llvm/llvm-3.2/tools/clang/test/PCH/missing-file.cpp 2> /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.stderr &&
grep 'could not find file' /home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.stderr
