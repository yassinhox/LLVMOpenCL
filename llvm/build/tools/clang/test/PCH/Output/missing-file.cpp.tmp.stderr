fatal error: malformed or corrupted PCH file: 'could not find file '/home/jazouani/llvm/build/tools/clang/test/PCH/Output/missing-file.cpp.tmp.h' referenced by AST file'
1 error generated.
